src/app/helpers/jwt.interceptor.spec.ts

```diff
@@ -1,45 +1,76 @@
-import {TestBed} from '@angular/core/testing';
-import {HttpClientTestingModule} from '@angular/common/http/testing';
-import {httpHandlerSpy, httpRequestSpy} from './spies';
-import {HttpHandler, HttpRequest} from '@angular/common/http';
-import {of} from 'rxjs';
-import {JwtInterceptor} from './jwt.interceptor';
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { httpHandlerSpy, httpRequestSpy } from './spies';
+import { HttpHandler, HttpRequest } from '@angular/common/http';
+import { of } from 'rxjs';
+import { JwtInterceptor } from './jwt.interceptor';
+import { AuthService } from '../services/auth.service';
```

Comment:
Let's make use of absolute imports.

Reply:
Thanks, this has been corrected.

Reply:
Hey @Manorlds-Eaglespark , this hasn't yet been fixed

Reply:
Please check again, thanks  @Tevinthuku